tools/conversion/convert_to_tflite.py

```diff
@@ -132,6 +138,9 @@ def convert(backbone_settings, classifier_settings, output_name, plot_model):
     float32 = args["--float32"]
     path_in = args["--path_in"]
     plot_model = args["--plot_model"]
+    verbose = args["--verbose"]
+    print(verbose)
```

Comment:
```suggestion
    print(f"Verbose log: {verbose})
```

Reply:
Oops, I actually didn't intend to commit that line :sweat_smile: